### PR TITLE
Small Fix for "Topologina Bambee"

### PR DESCRIPTION
**Pull request by Playmaker772211**
Should banish "Topologina" monsters only.

### CODE REVIEW

**Comment by gianni159** [comment]
Playmaker772211 if you have discord can you please add me gianni #9714

**Comment by AndreOliveiraMendes** [comment]
@Playmaker772211 keep the "LOCATION_MZONE+LOCATION_GRAVE" on line 21 because that line is necessary to it properly work with spirit elimination

**Comment by kevinlul** [comment]
Could you rename the PR to specify the name of the card? e.g. `Small fix for "Topologina Bambee"`

File: script/c511030050.lua [hunk 1]
```diff
@@ -15,10 +15,10 @@ function s.initial_effect(c)
 	c:RegisterEffect(e1)
 end
 function s.cfilter(c)
-	return c:IsSetCard(0x57b) and c:IsAbleToRemoveAsCost() and aux.SpElimFilter(c,true)
+	return c:IsSetCard(0x57b) and c:IsType(TYPE_MONSTER) and c:IsAbleToRemoveAsCost() and aux.SpElimFilter(c,true)
 end
 function s.spcost(e,tp,eg,ep,ev,re,r,rp,chk)
-	local rg=Duel.GetMatchingGroup(s.cfilter,tp,LOCATION_MZONE+LOCATION_GRAVE,0,nil)
+	local rg=Duel.GetMatchingGroup(s.cfilter,tp,LOCATION_GRAVE,0,nil)
```
Comment (kevinlul) [inline script/c511030050.lua]: Revert this change per @AndreOliveiraMendes's comment regarding the interaction with Spirit Elimination.

Comment (kevinlul) [inline script/c511030050.lua]: Revert this change per @AndreOliveiraMendes's comment regarding the interaction with Spirit Elimination.
